src/uwtools/rocoto.py

```diff
@@ -26,6 +48,67 @@ def _add_jobname(tree: dict) -> None:
 
 
 # Public functions
+def realize_rocoto_xml(
+    input_yaml: str,
+    rendered_output: str,
+) -> bool:  # pragma: no cover
+    """
+    Main entry point.
```

Comment:
It seems that  many of the functions in this module are now documented as "Main entry point". Would you mind making those more descriptive now that they've multiplied?